sql/procedures/create_override_publish_date_procedure.sql

```diff
@@ -0,0 +1,25 @@
+DELIMITER //
+DROP PROCEDURE IF EXISTS OverridePublishDates //
+CREATE PROCEDURE OverridePublishDates()
+BEGIN
+    UPDATE available_songs, publish_date_overrides
+    SET available_songs.publishedon = publish_date_overrides.override_data
+    WHERE available_songs.link = publish_date_overrides.video_id;
+    
+END //
+
+DROP TRIGGER IF EXISTS update_publish_overrides_trigger //
+CREATE TRIGGER update_publish_overrides_trigger
+AFTER UPDATE
+ON publish_date_overrides
+FOR EACH ROW
+CALL OverridePublishDates() //
+
+DROP TRIGGER IF EXISTS insert_publish_overrides_trigger //
+CREATE TRIGGER insert_publish_overrides_trigger
+AFTER INSERT
+ON publish_date_overrides
+FOR EACH ROW
+CALL OverridePublishDates() //
```

Comment:
Not too sure of how triggers work, but is it possible to only call this on the inserted entry instead of each row (if that's not what it's already doing?)

Reply:
It's possible to optimize this, but we'd need to have two separate procedures for updating all overrides and updating a single one. But it's not a very expensive call so didn't think it was worth doing

Reply:
Got it, thanks for clarifying